apps/bfd-pipeline/bfd-pipeline-app/pom.xml

```diff
@@ -98,7 +98,6 @@
 			<!-- JDBC driver for working with PostgreSQL DBs on Java 8+ (JDBC 4.2). -->
 			<groupId>org.postgresql</groupId>
 			<artifactId>postgresql</artifactId>
-			<scope>runtime</scope>
```

Comment:
👋 I've already approved, but I was curious about this specific line. What's going on here?

Reply:
well I needed to have access to that library at compile time as well....

Reply:
> well I needed to have access to that library at compile time as well....

Ah. It's just between compile time and runtime, then. That makes sense. Thanks!